engine/access/ingestion/engine.go

```diff
@@ -33,12 +33,18 @@ const collectionCatchupDBPollInterval = 10 * time.Millisecond
 const fullBlockUpdateInterval = 1 * time.Minute
 
 // a threshold of number of blocks with missing collections beyond which collections should be re-requested
+// this is to prevent spamming the collection nodes with request
 const missingCollsForBlkThreshold = 100
 
+// a threshold of block height beyond which collections should be re-requested (regardless of the number of blocks for which collection are missing)
+// this is to ensure that if a collection is missing for a long time (in terms of block height) it is eventually re-requested
+const missingCollsForHeightThreshold = 100
```

Comment:
a better name would be something like `missingCollAgeThreshold`

Reply:
changed to `missingCollAgeThreshold`